CHANGES.rst

```diff
@@ -4,6 +4,7 @@ Upcoming version
 Deprecated features
 -------------------
 - Replaced :code:`directory_input` keyword-argument of :code:`ManagedWindowBase` by :code:`enable_file_input` (@CasperSchippers, #964)
+- Deprecated to not specify :code:`incldueSCPI` for an instrument which uses SCPI (@BenediktBurger, #1007)
```

Comment:
```suggestion
- Deprecated to not specify :code:`includeSCPI` for an instrument which uses SCPI (@BenediktBurger, #1007)
```

Reply:
I don't understand your message, therefore, I made a new suggestion, which is, hopefully, easier to understand.

Reply:
No problem! However
```
:code:`incldueSCPI` obligarory for
```
-> 
```
:code:`includeSCPI` obligatory for
```

Reply:
Done. Thanks.